js/ui.js

```
@@ -1344,6 +1344,25 @@ export class UIRenderer {
+                const currentVolume = this.player.userVolume;
+                const newVolume = Math.max(0, Math.min(1, currentVolume + delta));
+
+                if (delta > 0 && this.player.muted) {
+                    this.player.setMute(false);
+                }
+
+                this.player.setVolume(newVolume);
+                updateFsVolumeUI();
+            };
+
+            [fsVolumeBar, fsVolumeBtn].forEach((el) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)